cost-analyzer/templates/kubecost-admission-controller-template.yaml

```diff
@@ -7,11 +7,6 @@ metadata:
 webhooks:
   - name: "kubecost-deployment-validation.kubecost.svc"
     failurePolicy: Ignore
-    namespaceSelector:
-      matchExpressions:
-        - key: kubernetes.io/metadata.name
-          operator: In
-          values: [ "kubecost" ]
```

Comment:
Does this mean that the Admission Controller will have permissions everywhere by default? If so, how are we clearly communicating security implications to our users?

Reply:
Given the possibly far-reaching implications, I'd like to see at least a sketch of documentation explaining this feature, how to think about security, and how to customize it before this merges.

I'm worried that documentation will fall through the cracks otherwise and I don't feel comfortable approving until I know we'll be communicating well with our users.

Reply:
Sure, want to take a look at https://docs.google.com/document/d/1yT0dhXaJ0T7Xw5LEvnDp4UG045y76bNjKTXy0jjoKmo/edit?usp=sharing ?

Reply:
That's great, thank you. I left one comment in that doc, but I feel comfortable approving this PR.